service/build.gradle

```diff
@@ -81,4 +84,23 @@ tasks.withType(Test) {
 	environment.put("GOOGLE_APPLICATION_CREDENTIALS", "${googleCredentialsFile}")
 }
 
-test { maxParallelForks = 1 }
+test {
+	maxParallelForks = 1
+	filter {
+		excludeTestsMatching "VerifyServicePacts*"
+	}
+}
+
+task verifyPacts( type: Test ) {
```

Comment:
this isn't being used anywhere in this PR. I have a followup PR that adds github actions that will run this task when verification is needed, but I want to keep it separate so that Ivan can help review in isolation.